.github/workflows/docs.yml

```diff
@@ -21,14 +21,40 @@ jobs:
         - name: Checkout code
           uses: actions/checkout@v2
           with:
-            fetch-depth: 0
+            submodules: 'true'
         - name: Set system to non-interactive mode
           run: export DEBIAN_FRONTEND=noninteractive
         - name: install dependencies
           run: |
             pip install sphinx
             pip install sphinx-rtd-theme
             pip install sphinx-multiversion
+            sudo apt install -y --force-yes -qq build-essential cmake
+        - name: generate csv files
+          run: |
+            echo "Repo = ${GITHUB_REPOSITORY}"
+            mkdir -p build && cd build
+            cmake -DPARTHENON_DISABLE_HDF5=ON \
+                  -DPARTHENON_DISABLE_MPI=ON \
+                  -DREGRESSION_GOLD_STANDARD_SYNC=OFF \
+                  -DBUILD_TESTING=OFF \
+                  ..
+            make -j4
+            make param-docs
+            cd ..
+        - name: Check generated files for this branch are up to date
+          run: |
+            if git diff --exit-code -- doc/sphinx/src/generated/; then
+              echo "Generated doc files up to date."
+            else
+              echo "Generated doc files not up to date. Run make param-docs and commit changes."
+              exit 1
+            fi
```

Comment:
We now build the code and run the executable with -p to generate the docs tables. Those docs tables are stored in version control because they may vary between branches. To ensure the correct version is stored, the CI compares against the committed version.

You can build them yourself and update them with `make param-docs`.